solutions/dotnet_azure_sdk/Identity_Resource_Management_and_Storage/src/Program.cs

```diff
@@ -42,37 +42,43 @@ static async Task<int> Main()
             // Azure.ResourceManager.Storage is currently in preview.
             var storageManagementClient = new StorageManagementClient(subscriptionId, credential);
 
-            try
+            int retries = 3; 
```

Comment:
I meant to repeat the query 3 times, and every time it should succeed.

It's not acceptable to retry 3 times to complete a query successfully. If that's the case for Mystikos, please file an issue.

Reply:
@jxyang Sorry, I misunderstood your previous comment.